### DISCUSSION TITLE
How to request arm64 bottle of Formula?

### DISCUSSION THREAD

**Discussion by Cyberbeni** [Homebrew/discussions#637] [Everyday usage] [answered]
There is this Formula: https://github.com/Homebrew/homebrew-core/blob/0557c9fbf41b50e19ac85ef76a595068f98a552a/Formula/mas.rb
I successfully installed it on ARM64 without any changes earlier this week. My guess is that it needed the recently released version of Carthage: https://github.com/Homebrew/homebrew-core/blob/e5b6b827579cac83b617ade406d7d2b8b538a38d/Formula/carthage.rb

Is there a way to request bottling without changing anything in the formula?

**Comment by SMillerDev** [comment, marked as answer]
You'll just have to wait until it either updates and we try an arm bottle, or someone on the homebrew team attempts to trigger a bottle. There is no way to request specific bottles.

**Reply by kitchen** [reply]
is there a way to mass trigger bottles just for funsies? There are a bunch of minor tools that I have that probably won't have much in the way of updates in the future that are currently missing from my Brewfile. I'm personally willing to do whatever work would be required to make these happen, I just am not sure what that work needs to be :)

**Reply by Cyberbeni** [reply]
```
brew install --build-from-source <formula>
brew test <formula>
```
where `<formula>` can also be a local file that you modified
You can just write a script that redirects output to /dev/null or a file and only prints the exit code of these

**Reply by kitchen** [reply]
right, I can install in manually locally, but I'd like to do whatever is necessary to make it accessible for all as well :)

**Comment by carlocab** [comment]
`mas` builds, but the test fails: https://github.com/Homebrew/homebrew-core/actions/runs/518978685

**Reply by Cyberbeni** [reply]
hmm, it also crashes on my machine which invalid code signature error and building it with verbose flag prints warnings at the end for install_name_tool invalidating the code signature. I'll have to look into that later.
